SECURITY_CONTACTS

```diff
@@ -0,0 +1,15 @@
+# Reporting a security vulnerability
+
+Please do:
+  - not disclose any security issue publicly e.g. Pull Requests, Comments.
+  - not disclose any security issue directly to any owner of the repository or
+    to any other contributor.
+
+In this repository security reports are handled according to the
+Metal3-io project's security policy. For more information about the security
+policy consult the User-Guide [here](https://metal3io.netlify.app/security_policy.html).
```

Comment:
```suggestion
In this repository, security reports are handled according to the
Metal3-io project's security policy. For more information about the security
policy consult the User Guide [here](https://metal3io.netlify.app/security_policy.html).
```

Reply:
We are referring to it as User-guide or user-guider in many places https://github.com/metal3-io/metal3-docs/tree/main/docs bot in directory names and in text